src/lib/y2storage/proposal_settings.rb

```diff
@@ -401,6 +402,14 @@ def apply_defaults
       end
     end
 
+    # Some values can be explicitly enforced by user
+    # This setting should have precendence over everything else
+    def apply_user_enforced
+      value = StorageEnv.instance.reuse_lvm?
```

Comment:
Now I miss the `require "y2storage/storage_env"` at the top of this file.

Reply:
usual consequence of moving code here and there :-/ I have to test run it again.